_layouts/tour.html

```diff
@@ -16,29 +16,40 @@ <h1>2022 载歌在谷中秋游园</h1>
         </div>
     </div>
 </div>
-<div class="container">
-    <div class="row pt-10">
-        <div class="col">
-            <h1>{{page.title}}</h1>
+
+<!-- Tab links -->
+<div class="tab">
+    <button class="tab-links" onclick="openTab(event, 'tour-list')" id="defaultOpen">2023春节游园会活动介绍</button>
+    <button class="tab-links" onclick="openTab(event, 'sponsor')">赞助商</button>
```

Comment:
1. Since we might have multiple tab groups on one page, `default-open` should be a class, rather than an id.
2. `defaultOpen` -> `default-open`.
3. You can make use of [data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) on HTML elements to pass arguments to the callback function.

```suggestion
    <button class="tab-links default-open" onclick="openTab(event)" data-tab-group="main" data-tab-id="tour-list">2023春节游园会活动介绍</button>
    <button class="tab-links" onclick="openTab(event)" data-tab-group="main" data-tab-id="sponsor">赞助商</button>
```

Reply:
Updated.